pkg/controller/federatedtypeconfig/controller.go

```diff
@@ -288,6 +289,14 @@ func (c *Controller) startController(tc *corev1a1.FederatedTypeConfig) error {
 		return fmt.Errorf("Error starting sync controller for %q: %v", kind, err)
 	}
 	glog.Infof("Started sync controller for %q", kind)
+	if tc.GetStatus() != nil {
```

Comment:
enabling/disabling of status can be achieved by including/excluding `status:` section in FederatedTypeConfig. 
Are you suggesting we should disable by default? i agree with that.
if you prefer to add another field like `enableStatus: false`. i am fine with that too.
which one do you prefer?

Reply:
I think it would be desirable to allow status to be enabled or disabled in the same way as propagation (e.g. `enableStatus`).  My thinking is that it avoids the user having to add back the status configuration vs just flipping a boolean flag.